string_challenges.py

```diff
@@ -1,28 +1,39 @@
 # Вывести последнюю букву в слове
 word = 'Архангельск'
-# ???
+print(word[-1])
 
 
 # Вывести количество букв "а" в слове
 word = 'Архангельск'
-# ???
+word = word.lower()
+print(word.count('а'))
 
 
 # Вывести количество гласных букв в слове
+rus_vowels = 'аоуэыяёеюи'
 word = 'Архангельск'
-# ???
+word_vowels_cnt = 0
+for letter in word.lower():
+    if letter in rus_vowels:
+        word_vowels_cnt += 1
```

Comment:
Слегка спорное решение, но мне нравится. Можно добавлять bool в инкременте, True=1, False=0
На твое усмотрение
```suggestion
        word_vowels_cnt += letter in rus_vowels
```